README.md

```diff
@@ -36,8 +36,12 @@ This group's review is mandated on all API changes, and their goal is twofold:
  
 Even changes that seem trivial can often be made more consistent and future-proof with some modifications, and the folks on the API WG have the expertise to spot and suggest those changes.
 
-In accordance with the above goals, this bot performs several bookkeeping duties. When a new PR is opened which is either `semver/minor` or `semver/major`, it will automatically add an `api-review/requested 🗳` label to the PR. To add clarity to whether a review is occurring in a given Electron governance member's capacity as a member of the API WG, this bot then adds a GitHub Check on the PR which members can click on to affirm their review choices:
+In accordance with the above goals, this bot performs several bookkeeping duties. When a new PR is opened which is either `semver/minor` or `semver/major`, it will automatically add an `api-review/requested 🗳` label to the PR. To add clarity to whether a review is occurring in a given Electron governance member's capacity as a member of the API WG, this bot then adds a GitHub Check on the PR which members can click on to affirm their review choices.
```

Comment:
```suggestion
In accordance with the above goals, this bot performs several bookkeeping duties. When a new PR is opened which is either `semver/minor` or `semver/major`, it will automatically add an `api-review/requested 🗳` label to the PR.
```

Reply:
@jkleinsc it still does add a check - i can remove 
> on the PR which members can click on to affirm their review choices.

but i think we should still note that ready-status will show up in the GitHub Checks list for clarity!